apps/vite/src/components/SingleCalendarEvent.tsx

```
@@ -1,30 +1,33 @@
-import { relativeTimeFromDates } from "../helper/dates"
-import nepaliNumber from "../helper/nepaliNumber"
-import { Event } from "@miti/types"
-import UseLanguage from "../helper/useLanguage"
-import NepaliDate from "nepali-datetime"
+import NepaliDate from 'nepali-datetime';
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)